packages/data-context/src/sources/RelevantRunsDataSource.ts

```diff
@@ -181,16 +181,24 @@ export class RelevantRunsDataSource {
   /**
    * Clear the cached current run to allow the data source to pick the next completed run as the current
    */
-  async moveToNext (shas: string[]) {
+  async moveToNext (runNumber: number, shas: string[]) {
```

Comment:
Should this still be named `moveToNext`? Seems like it's more of a `moveToRun` now